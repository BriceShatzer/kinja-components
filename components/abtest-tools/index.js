/* @flow */

import ABTestIframes from './abtest-iframes';

export default ABTestIframes;
