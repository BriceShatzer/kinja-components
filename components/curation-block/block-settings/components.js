// @flow
import styled from 'styled-components';
import media from '../../../style-utils/media';
import { gridValue } from '../../grid-utils';
import { Wrapper as RadioWrapper } from '../../form/radio';

export const SectionHeader = styled.h4`
	color: ${props => props.theme.color.darkgray};
	font-size: ${props => props.theme.typography.primary.fontSizes.small};
	line-height: ${props => props.theme.typography.primary.lineHeights.small};
	margin-bottom: 1.5rem;
`;

export const FieldHeaderLabel = styled.label`
	color: ${props => props.theme.color.gray};
	font-size: ${props => props.theme.typography.primary.fontSizes.xsmall};
	line-height: ${props => props.theme.typography.primary.lineHeights.xsmall};
	margin-bottom: 1.5rem;
`;

export const Container = styled.div`
	margin-bottom: 2rem;
`;

export const ColumnWrapper = styled.div`
	${media.xlargeUp`
		display: flex;
	`}
`;

export const Column = styled.div`
	width: ${gridValue.small('6c')};
	margin-bottom: 1rem;
	${media.xlargeUp`
		margin-bottom: 0;
	`}

	${media.xlargeOnly`
		width: ${gridValue.xlarge('3c')};
		&:not(:last-child) {
			margin-right: ${gridValue.xlarge('1g')};
		}
	`}

	${media.xxlargeOnly`
		width: ${gridValue.xxlarge('3c')};
		&:not(:last-child) {
			margin-right: ${gridValue.xxlarge('1g')};
		}
	`}

	${media.xxxlargeUp`
		width: ${gridValue.xxxlarge('3c')};
		&:not(:last-child) {
			margin-right: ${gridValue.xxxlarge('1g')};
		}
	`}

	${RadioWrapper} {
		height: auto;
	}
`;