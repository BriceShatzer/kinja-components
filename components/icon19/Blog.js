// @flow

import * as React from 'react';

import Icon19 from './icon19';
import type { Icon19Type } from './icon19';
import Blog from './svg/Blog/Blog.svg';

const BlogIcon = (props: Icon19Type) =>
	<Icon19 {...props}>
		<Blog/>
	</Icon19>;

export default BlogIcon;
