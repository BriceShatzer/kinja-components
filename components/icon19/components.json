{
  "18x18": [
    "AddBlog",
    "AlignBottomCenter",
    "AlignBottomLeft",
    "AlignBottomRight",
    "AlignCenter",
    "AlignLeft",
    "AlignMiddleCenter",
    "AlignRight",
    "ArrowBottomLeft",
    "ArrowBottomRight",
    "ArrowBottom",
    "ArrowLeft",
    "ArrowRight",
    "ArrowTopLeft",
    "ArrowTopRight",
    "ArrowTop",
    "Bell",
    "Bleed",
    "Blog",
    "Bold",
    "Bookmark",
    "BookmarkFilled",
    "Breaking",
    "Bubble",
    "BubbleAdd",
    "BubbleDelete",
    "BubbleFilled",
    "BubbleRemove",
    "Burner",
    "Cart",
    "CartButton",
    "Center",
    "Checkmark",
    "ChevronDown",
    "ChevronLeft",
    "ChevronRight",
    "ChevronUp",
    "Clock",
    "Close",
    "Code",
    "Collapse",
    "ColorPicker",
    "Compare",
    "Copy",
    "Copyright",
    "Crop",
    "CurationTertiaryFour",
    "CurationTertiaryOne",
    "Disabled",
    "DollarCircle",
    "Dollar",
    "DoubleChevronDown",
    "DoubleChevronLeft",
    "DoubleChevronRight",
    "DoubleChevronUp",
    "DropCap",
    "Editor",
    "Ellipsis",
    "EmbedThumb",
    "Embiggen",
    "EmbiggenFilled",
    "ErrorFilled",
    "Error",
    "Expand",
    "ExpandBackground",
    "ExternalLink",
    "ExternalLinkError",
    "Eye",
    "EyeFilled",
    "Facebook",
    "FeaturedHeaderClassic",
    "FeaturedHeaderClassicReversed",
    "FeaturedHeaderImpact",
    "FeaturedHeaderMagazine",
    "Flag",
    "Flame",
    "Flask",
    "Folder",
    "FolderVideo",
    "FullBleedWidget",
    "Fullscreen",
    "Gear",
    "Globe",
    "Google",
    "GoogleColor",
    "Hamburger",
    "Hashmark",
    "Heart",
    "Home",
    "HorizontalRule",
    "Image",
    "Inbox",
    "Information",
    "InsertModule",
    "InsetMinimal",
    "Instagram",
    "Issue",
    "Italic",
    "Key",
    "Kinja",
    "Label",
    "LayoutHorizontal",
    "LayoutVertical",
    "Left",
    "Lightning",
    "Link",
    "Linkedin",
    "Loading",
    "Lock",
    "LockFilled",
    "Login",
    "Logout",
    "Mail",
    "MailFilled",
    "Minus",
    "MirrorHorizontal",
    "MirrorVertical",
    "MomJeans",
    "MoveBottom",
    "MoveLeft",
    "MoveRight",
    "MoveTop",
    "MoveVertical",
    "OrderNewest",
    "OrderOldest",
    "OrderPopular",
    "OrderedList",
    "OverlayDark",
    "OverlayLight",
    "Pagebreak",
    "Parallax",
    "Pause",
    "PauseFilled",
    "Pencil",
    "Pin",
    "Play",
    "PlayFilled",
    "Plus",
    "Promotion",
    "Quotable",
    "Quote",
    "RSS",
    "RecommendedStories",
    "Reload",
    "ReplyLeft",
    "ReplyRight",
    "Review",
    "Right",
    "Rotate",
    "Search",
    "SearchVideo",
    "Settings",
    "Share",
    "SharedPost",
    "Slideshow",
    "SortAlphabetically",
    "Star",
    "StarCircle",
    "StarFilled",
    "Strikethrough",
    "TextColorDark",
    "TextColorLight",
    "TileBackground",
    "Trashcan",
    "Twitter",
    "Redo",
    "Undo",
    "UnorderedList",
    "User",
    "UserAdd",
    "UserAvatar",
    "UserFilled",
    "UserRemove",
    "Users",
    "Video",
    "VolumeMax",
    "Write",
    "Youtube",
    "ZoomOut",
    "ZoomIn"
  ],
  "24x24": [
    "Blog24",
    "Bookmark24",
    "BookmarkFilled24",
    "Bubble24",
    "Clock24",
    "Close24",
    "DealsBox24",
    "Flame24",
    "Hamburger24",
    "Label24",
    "Promotion24",
    "Search24",
    "SharedPost24",
    "Tools24",
    "Video24"
  ],
  "48x48": [
    "CurationEqualFour",
    "CurationEqualSix",
    "CurationEqualThree",
    "CurationHeadline",
    "CurationModularFour",
    "CurationModularThree",
    "Curation20Ad",
    "Curation20EightCard",
    "Curation20FeaturedStory",
    "Curation20FiveCard",
    "Curation20FiveCardTwoColumn",
    "Curation20FourCard",
    "Curation20FourCardHorizontal",
    "Curation20FourCardHorizontalCompact",
    "Curation20SingleStory",
    "Curation20SixCardGrid",
    "Curation20SixCardWithLatest",
    "Curation20ThreeCardHorizontal",
    "Curation20Video",
    "DealsBeauty",
    "DealsGaming",
    "DealsHome",
    "DealsLifestyle",
    "DealsMultimedia",
    "DealsTech",
    "DealsTop",
    "LunchboxHeadline",
    "LunchboxHeadlineHero",
    "LunchboxHeadlineImage",
    "LunchboxHeadlineText",
    "LunchboxHeadlineTextHero",
    "LunchboxHeadlineTextImage",
    "LunchboxHeroHeadline",
    "LunchboxHeroHeadlineText",
    "LunchboxHeroText",
    "LunchboxImage",
    "LunchboxImageFullbleed",
    "LunchboxImageHeadline",
    "LunchboxImageHeadlineText",
    "LunchboxImageParallax",
    "LunchboxImageText",
    "LunchboxText",
    "LunchboxTextHero",
    "LunchboxTextImage"
  ]
}