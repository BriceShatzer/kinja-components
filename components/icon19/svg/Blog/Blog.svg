<svg xmlns="http://www.w3.org/2000/svg" width="18" height="18" aria-label="Blog icon" viewBox="0 0 18 18">
	<path fill-rule="evenodd" d="M17 14.5a2.5 2.5 0 0 1-2.5 2.5h-11A2.5 2.5 0 0 1 1 14.5v-11A2.5 2.5 0 0 1 3.5 1h11A2.5 2.5 0 0 1 17 3.5v11zM16 5H2v9.5c0 .83.67 1.5 1.5 1.5h11c.83 0 1.5-.67 1.5-1.5V5zm-1.5-3h-11C2.66 2 2 2.67 2 3.5V4h14v-.5c0-.83-.67-1.5-1.5-1.5zM5 12v1h1v-1H5zm-1 0a1 1 0 0 1 1-1h1a1 1 0 0 1 1 1v1a1 1 0 0 1-1 1H5a1 1 0 0 1-1-1v-1zm4.5 0a.5.5 0 1 1 0-1h5a.5.5 0 1 1 0 1h-5zm0 2a.5.5 0 1 1 0-1h3a.5.5 0 1 1 0 1h-3zM5 8v1h1V8H5zM4 8a1 1 0 0 1 1-1h1a1 1 0 0 1 1 1v1a1 1 0 0 1-1 1H5a1 1 0 0 1-1-1V8zm4.5 0a.5.5 0 0 1 0-1h5a.5.5 0 1 1 0 1h-5zm0 2a.5.5 0 0 1 0-1h3a.5.5 0 1 1 0 1h-3z"/>
</svg>
