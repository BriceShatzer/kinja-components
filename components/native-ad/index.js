/* @flow */

export { default as StreamNativeAd } from './stream/stream-native-ad';
export { default as LeftRailNativeAd } from './left-rail/left-rail-native-ad';
export { default as RecircNativeAd } from './recirc/recirc-native-ad';