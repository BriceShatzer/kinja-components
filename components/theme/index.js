// @flow

import Theme from './theme';
import { matchBlogGroupToTheme } from './theme';
import defaultTheme from './themes';
import EnsureDefaultTheme from './ensureDefaultTheme';

export {
	Theme,
	defaultTheme,
	EnsureDefaultTheme,
	matchBlogGroupToTheme
};
export default Theme;
