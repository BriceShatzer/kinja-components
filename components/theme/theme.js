// @flow

import * as React from 'react';
import { ThemeProvider } from 'styled-components';

import { blogThemes as Themes } from './themes';

export type BlogThemeName = $Keys<typeof Themes>;
const blogThemeNames: Array<BlogThemeName> = Object.keys(Themes);

type Props = {
	children: React.Node,
	theme: {},
	blog?: ?BlogThemeName
};

export const matchBlogGroupToTheme = (blogGroup: ?string): BlogThemeName => {
	return (blogThemeNames.find(b => b === blogGroup)) || 'default';
};

const Theme = (props: Props) => {
	const { children, theme, blog } = props;

	const appliedTheme = blog ? {
		...(Themes[blog] || Themes.default),
		blog
	} : theme;

	return (
		<ThemeProvider theme={appliedTheme}>
			{children}
		</ThemeProvider>
	);
};

Theme.defaultProps = {
	theme: Themes.default
};

export default Theme;
