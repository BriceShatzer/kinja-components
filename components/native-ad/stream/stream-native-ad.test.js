import * as React from 'react';
import { render } from 'enzyme';

import StreamNativeAd from './stream-native-ad';
import NativeAd from 'kinja-magma/models/NativeAd';
import Blog from 'kinja-magma/models/Blog';
import { EnsureDefaultTheme } from 'kinja-components/components/theme';

import nativeAds from '../__fixtures__/native-ads';
import blog from '../../../__stubs__/gizmodo.json';

const currentPost = nativeAds.map(post => NativeAd.fromJSON(post))[0];
const currentBlog = Blog.fromJSON({
	...blog,
	...{
		properties: {
			...blog.properties
		}
	}
});

describe('Stream Native Ad', () => {
	it('should statically render a stream native ad', () => {
		const wrapper = render(
			<EnsureDefaultTheme>
				<StreamNativeAd
					index={0}
					post={currentPost}
					blog={currentBlog}
				/>
			</EnsureDefaultTheme>
		);

		expect(wrapper).toMatchSnapshot();
	});
});