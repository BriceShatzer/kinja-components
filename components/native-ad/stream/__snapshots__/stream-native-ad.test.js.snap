// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stream Native Ad should statically render a stream native ad 1`] = `
.c16 {
  text-align: left;
  font-size: 24px;
  font-family: ProximaNovaCond,sans-serif;
  line-height: 1.3;
  margin-top: 0;
  margin-bottom: 0;
}

.c11 {
  -webkit-text-decoration: none;
  text-decoration: none;
}

.c9 {
  float: left;
  margin: 0;
  position: relative;
}

.c9 .c18 {
  border: 2px solid #FFF;
  left: 0.5rem;
  top: 0.5rem;
  font-size: 1.125rem;
  line-height: 1.125rem;
  width: 2rem;
  height: 2rem;
}

.c12 {
  background-color: #F5F5F5;
  background-position: center;
  background-repeat: no-repeat;
  background-size: cover;
  display: block;
  height: 0;
  overflow: hidden;
  padding-bottom: 56.25%;
  position: relative;
  width: 100%;
}

.c12 img {
  background: #F5F5F5;
  overflow: hidden;
}

.c12 img.lazyloaded {
  background: none;
}

.c12 img.ls-pre-lazy-img {
  height: 100%;
  background: none;
}

.c12 video {
  max-width: 100%;
  height: auto;
}

.c12 video::-webkit-media-controls-start-playback-button {
  display: none;
  -webkit-appearance: none;
}

.c12 img,
.c12 video {
  height: auto;
  width: 100%;
}

.c5 {
  font-family: ProximaNovaCond,sans-serif;
  font-size: 15px;
  font-weight: bold;
  line-height: 25px;
  text-transform: uppercase;
  color: #222;
}

.c8 {
  fill: currentColor;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c7 {
  margin-right: 6px;
  margin-top: -2px;
}

.c7.right {
  margin-right: 0;
  margin-left: 10px;
}

.c6 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  color: #222;
  font-family: ProximaNovaCond,sans-serif;
  line-height: 25px;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c1 {
  position: relative;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
}

.c2 .c3:not(:empty) {
  margin-bottom: 4px;
}

.c14 .c3:not(:empty) {
  margin: -1px 0 5px;
}

.c15 .c10 {
  color: #000;
}

.c15 .c10:hover {
  color: #000;
}

.c17 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  margin-top: -4px;
}

.c0 {
  padding-left: 1.125rem;
}

@media only screen and (min-width:37.37em) {
  .c16 {
    margin-bottom: 8px;
  }
}

@media only screen and (min-width:37.37em) {
  .c9 {
    -webkit-flex: 1;
    -ms-flex: 1;
    flex: 1;
    margin-right: 19px;
    max-width: 247px;
  }
}

@media only screen and (max-width:37.31em) {
  .c9 {
    margin-bottom: 8px;
  }
}

@media only screen and (max-width:49.94em) {
  .c4 {
    -webkit-flex-direction: row;
    -ms-flex-direction: row;
    flex-direction: row;
  }
}

@media only screen and (max-width:37.31em) {
  .c1 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

@media only screen and (min-width:37.37em) {
  .c2 {
    display: none;
  }
}

@media only screen and (max-width:37.31em) {
  .c14 {
    display: none;
  }
}

@media only screen and (min-width:37.37em) {
  .c13 {
    -webkit-flex: 2.162;
    -ms-flex: 2.162;
    flex: 2.162;
  }
}

@media only screen and (max-width:37.31em) {
  .c17 {
    margin-top: 10px;
  }
}

@media only screen and (min-width:37.37em) {
  .c0 {
    padding-left: 1.125rem;
    padding-right: calc(82px + 1.125rem);
  }
}

<div
  class="c0"
>
  <article
    class="js_post_item c1"
    data-commerce-source="nativestream"
    data-id="0"
    data-index="0"
  >
    <div
      class="c2"
    >
      <div
        class="c3 c4"
      >
        <span
          class="c5"
        >
          Promoted
        </span>
        <div
          class="c6"
        >
          <span
            class="c7"
            id="externalIcon"
          >
            <span
              class="c8"
            >
              <svg
                aria-label="ExternalLink icon"
                height="18"
                viewBox="0 0 18 18"
                width="18"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M6 3a.5.5 0 0 1 0 1H3.5C2.67 4 2 4.67 2 5.5v9c0 .83.67 1.5 1.5 1.5h9c.83 0 1.5-.67 1.5-1.5V11a.5.5 0 1 1 1 0v3.5a2.5 2.5 0 0 1-2.5 2.5h-9A2.5 2.5 0 0 1 1 14.5v-9A2.5 2.5 0 0 1 3.5 3H6zm10.51-2h.06l.04.01.04.01.04.02a.5.5 0 0 1 .15.1l.03.02.01.02h.01l-.03-.02.03.03.03.04a.52.52 0 0 1 .08.27v6a.5.5 0 1 1-1 0V2.7l-8.15 8.15a.5.5 0 0 1-.7-.7L15.29 2H10.5a.5.5 0 1 1 0-1h6.01z"
                  fill-rule="evenodd"
                />
              </svg>
            </span>
          </span>
          <span
            class=""
          >
            View on The Inventory
          </span>
        </div>
      </div>
    </div>
    <figure
      class="c9"
    >
      <a
        class="js_link c10 c11"
        data-ga="[[\\"Front page click\\",\\"stream post click - 0\\",\\"https://theinventory.com/ride-out-the-winter-in-sherpa-and-shearling-lined-every-1831468858%3Futm_source%3Ddfpnative%26utm_medium%3Dhomepage%26utm_campaign%3Dnative_sherpasherling_ds\\",{\\"metric19\\":1}]]"
        data-kala="[\\"STREAM_CLICK\\",\\"POST\\",\\"0\\"]"
        href="https://theinventory.com/ride-out-the-winter-in-sherpa-and-shearling-lined-every-1831468858%3Futm_source%3Ddfpnative%26utm_medium%3Dhomepage%26utm_campaign%3Dnative_sherpasherling_ds"
      >
        <div
          class="js_lazy-image c12"
        >
          <div>
            <img
              data-anim-src=""
              data-chomp-id="wvym61kyx8vht0kremsu"
              data-format="jpg"
              draggable="auto"
              src="data:image/gif;base64,R0lGODlhAQABAAAAACH5BAEKAAEALAAAAAABAAEAAAICTAEAOw=="
              srcset="https://i.kinja-img.com/gawker-media/image/upload/c_fill,f_auto,fl_progressive,g_center,h_180,q_80,w_320/wvym61kyx8vht0kremsu.jpg 320w, https://i.kinja-img.com/gawker-media/image/upload/c_fill,f_auto,fl_progressive,g_center,h_264,q_80,w_470/wvym61kyx8vht0kremsu.jpg 470w, https://i.kinja-img.com/gawker-media/image/upload/c_fill,f_auto,fl_progressive,g_center,h_80,q_80,w_80/wvym61kyx8vht0kremsu.jpg 80w"
            />
          </div>
        </div>
      </a>
    </figure>
    <div
      class="c13"
    >
      <div
        class="c14"
      >
        <div
          class="c3 c4"
        >
          <span
            class="c5"
          >
            Promoted
          </span>
          <div
            class="c6"
          >
            <span
              class="c7"
              id="externalIcon"
            >
              <span
                class="c8"
              >
                <svg
                  aria-label="ExternalLink icon"
                  height="18"
                  viewBox="0 0 18 18"
                  width="18"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M6 3a.5.5 0 0 1 0 1H3.5C2.67 4 2 4.67 2 5.5v9c0 .83.67 1.5 1.5 1.5h9c.83 0 1.5-.67 1.5-1.5V11a.5.5 0 1 1 1 0v3.5a2.5 2.5 0 0 1-2.5 2.5h-9A2.5 2.5 0 0 1 1 14.5v-9A2.5 2.5 0 0 1 3.5 3H6zm10.51-2h.06l.04.01.04.01.04.02a.5.5 0 0 1 .15.1l.03.02.01.02h.01l-.03-.02.03.03.03.04a.52.52 0 0 1 .08.27v6a.5.5 0 1 1-1 0V2.7l-8.15 8.15a.5.5 0 0 1-.7-.7L15.29 2H10.5a.5.5 0 1 1 0-1h6.01z"
                    fill-rule="evenodd"
                  />
                </svg>
              </span>
            </span>
            <span
              class=""
            >
              View on The Inventory
            </span>
          </div>
        </div>
      </div>
      <div
        class="c15"
      >
        <a
          class="js_link c10 c11"
          data-ga="[[\\"Front page click\\",\\"stream post click - 0\\",\\"https://theinventory.com/ride-out-the-winter-in-sherpa-and-shearling-lined-every-1831468858%3Futm_source%3Ddfpnative%26utm_medium%3Dhomepage%26utm_campaign%3Dnative_sherpasherling_ds\\",{\\"metric19\\":1}]]"
          data-kala="[\\"STREAM_CLICK\\",\\"POST\\",\\"0\\"]"
          href="https://theinventory.com/ride-out-the-winter-in-sherpa-and-shearling-lined-every-1831468858%3Futm_source%3Ddfpnative%26utm_medium%3Dhomepage%26utm_campaign%3Dnative_sherpasherling_ds"
        >
          <h2
            class="c16"
          >
            Ride Out the Winter In Sherpa and Shearling-Lined Everything
          </h2>
        </a>
      </div>
      <div
        class="c17"
      />
    </div>
  </article>
</div>
`;
